README.md

```diff
@@ -258,6 +273,7 @@ Arguments:
 - name: The name of the publication
 - collection: A function that returns the collection to publish (or just a collection, if it's insecure).
 - selector: (Optional) A function that returns mongo selector that will limit the results published (or just the selector).
+- settings: (Optional) A object with settings on server side's publish function. (Details see below)
```

Comment:
I have done a local test by mixing client and server side options (*new). Seems to work.

Without that I would call it a security problem, cause right now you expose the complete collection with all fields to the client. A user using the in browser console could see whatever he wants. In code (server) is a little ._extends(options, settings.options) that will overwirte those, declared on the server (e.g. to limit fields). 

So the server settings look like:

~~~javascript
ReactiveTable.publish(...

, {
enableRegex: true,
options: {
  fields: {
   secret: 0
  }
}
~~~